engine/src/main/java/org/entando/entando/aps/system/init/DatabaseManager.java

```diff
@@ -635,4 +661,13 @@ public void setDefaultDataSources(List<DataSource> defaultDataSources) {
     public void setLockFallbackMinutes(int lockFallbackMinutes) {
         this.lockFallbackMinutes = lockFallbackMinutes;
     }
+
+    protected ITenantManager getTenantManager() {
+        return tenantManager;
+    }
+    @Autowired
```

Comment:
is it possible to move the Autowire in the constructor?  .... it's the best practice ...

Reply:
Removed Autowired